src/main/java/org/parabot/core/Configuration.java

```diff
@@ -11,8 +11,10 @@
     public static final String LOGIN_SERVER             = "http://bdn.parabot.org/api/v2/users/login";
     public static final String GET_SCRIPTS              = "http://bdn.parabot.org/api/get.php?action=scripts_scripts&server=";
     public static final String GET_SCRIPT               = "http://bdn.parabot.org/api/get.php?action=scripts_script&id=";
+    public static final String GET_SERVER_PROVIDER_TYPE = "http://v3.bdn.parabot.org/api/bot/server/type?server=";
```

Comment:
`server=%s` so you can apply string format to this one too (makes the constant more readable)